packages/valory/skills/abstract_abci/handlers.py

```diff
@@ -52,7 +53,9 @@ class ABCIHandler(Handler):
 
     def setup(self) -> None:
         """Set up the handler."""
-        self.context.logger.debug("ABCI Handler: setup method called.")
+        self.context.logger.debug(
```

Comment:
Although the code is not used, they are tightly coupled and so for now I demo this in that way: